src/OutputWriters/checkpointer.jl

```diff
@@ -93,23 +93,28 @@ defaultname(::Checkpointer, nelems) = :checkpointer
 convert_to_arch(::CPU, a) = a
 convert_to_arch(::GPU, a) = CuArray(a)
 
-function restore_if_not_missing(file, location)
-    if haskey(file, location)
-        return file[location]
+function restore_if_not_missing(file, address)
+    if haskey(file, address)
+        return file[address]
     else
-        @warn "Checkpoint file does not contain $location. Returning missing. " *
+        @warn "Checkpoint file does not contain $address. Returning missing. " *
               "You might need to restore this manually."
         return missing
     end
 end
 
-function restore_field(file, location, arch, grid)
-    field_location = file[location * "/location"]
-    data = OffsetArray(convert_to_arch(arch, file[location * "/data"]), grid)
-    bcs = restore_if_not_missing(file, location * "/boundary_conditions")
-    return Field(field_location, arch, grid, bcs, data)
+function restore_field(file, address, arch, grid, loc)
+    field_address = file[address * "/address"]
+    data = OffsetArray(convert_to_arch(arch, file[address * "/data"]), grid, loc)
+    bcs = restore_if_not_missing(file, address * "/boundary_conditions")
+    return Field(field_address, arch, grid, bcs, data)
```

Comment:
```suggestion
    return Field(field_location, arch, grid, bcs, data)
```

Reply:
Hmm... I'm confused. I changed the name of this variable from `location` to `address` to prevent confusion with the "location tuple" that expresses staggered grid information, eg a tracer field has location `(Cell, Cell, Cell)`.

Reply:
Ok, I see your comment above.